content/en/docs/tasks/tools/install-kubectl.md

```diff
@@ -12,7 +12,7 @@ card:
 ---
 
 {{% capture overview %}}
-Use the Kubernetes command-line tool, [kubectl](/docs/user-guide/kubectl/), to deploy and manage applications on Kubernetes. Using kubectl, you can inspect cluster resources; create, delete, and update components; look at your new cluster; and bring up example apps.
+The Kubernetes command-line tool, [kubectl](/docs/user-guide/kubectl/), allows you to deploy and manage applications on Kubernetes. Using kubectl, you can inspect cluster resources; create, delete, and update components; look at your new cluster; and bring up example apps.
```

Comment:
The section sentence "Using kubectl, you can inspect cluster resources; create, delete, and update components; look at your new cluster; and bring up example apps." can be rewritten a bit better.